src/looprestoration_tmpl_8.rs

```diff
@@ -1037,7 +1061,77 @@ unsafe extern "C" fn loop_restoration_dsp_init_x86(
         (*c).sgr[2] = Some(dav1d_sgr_filter_mix_8bpc_avx512icl);
     }
 }
-
+// This calculates things slightly differently than the reference version.
+// This version calculates roughly this:
+// int32_t sum = 0;
+// for (int i = 0; i < 7; i++)
+//     sum += mid[idx] * fv[i];
+// sum = (sum + rounding_off_v) >> round_bits_v;
+// This function assumes that the width is a multiple of 8.
+#[cfg(all(feature = "asm", target_arch = "arm"))]
+unsafe extern "C" fn wiener_filter_neon(
+    dst: *mut pixel,
+    stride: ptrdiff_t,
+    left: *const [pixel; 4],
+    mut lpf: *const pixel,
+    w: libc::c_int,
+    h: libc::c_int,
+    params: *const LooprestorationParams,
+    edges: LrEdgeFlags,
+) {
+    let filter: Align16<[[int16_t; 8]; 2]> = (*params).filter;
```

Comment:
`filter` isn't aligned in the C version, and is a ptr to `[i16; 8]`, not an array of it.  So this should be a reference/pointer to `(*params).filter`, not a copy of its array.  Same for the 16 bpc one.